icgcget/cli.py

```diff
@@ -32,14 +32,15 @@
 def config_parse(filename):
 
     try:
-        config_text = open(filename, 'r')
+        config_text = filename.read()
+        filename.close()
     except IOError:
-        print("Config file " + filename + " not found: Aborting")
+        print("Config file " + filename.name + " not found: Aborting")
         sys.exit(1)
     try:
         config_temp = yaml.load(config_text)
     except yaml.YAMLError:
-        print("Could not read config file" + filename + ": Aborting")
+        print("Could not read config file" + filename.name + ": Aborting")
```

Comment:
Print is only used on errors that occur before the logger can be instantiated:namely being unable to find the config or unable to write to the log file